backend/doc/setting.md

```diff
@@ -192,6 +192,15 @@ For instance:
 
 The last one will check the internal user database first. Then, if the auth fails, it will also try to authenticate the user using LDAP.
 
+The authentication can be done with a login. A login can be either the user's email address or the username. In case there is
+no clear distinction in the authentication method used, login with "@" will be considered as email and login without as username.
```

Comment:
```suggestion
The authentication is done with a login, which is either the user's email address or the username. For authentication methods which don't provide a way to distinguish between the two, a login with "@" will be considered as an email address and a login without as username.
```

@inkhey what about usernames typed as `@someone` *?

Reply:
This is not supported in backend now
In current implementation, it's the frontend that does the jobs to deal with leading "@". We can handle this in backend but that either mean handled it deeply in tracimlib which is inconsistent with the fact that frontend does the job of removing it, or this mean specific code for each auth method (which is not nice).

I don't think it's a required feature to handle this for technical auth mecanism.

Reply:
By the way, there was a discussion about this here: https://github.com/tracim/tracim/issues/3061